### PR TITLE
[tools] Add python64 bit.

### PR DESCRIPTION
**Pull request by JackBoosY**
1. Add 64 bit python2/python3.
2. Choose the different python version depending on the architecture.

Related: #7411.

### CODE REVIEW

**Comment by JackBoosY** [comment]
@cbezault

**Comment by Rastaban** [comment]
https://dev.azure.com/vcpkg/public/_build/results?buildId=14308

**Comment by JackBoosY** [comment]
@grdowns can you restart CI and test all ports? 
This time is very busy, so this PR is not processed, and the last CI result has expired.

**Comment by JackBoosY** [comment]
@grdowns Consider to merge this PR?

**Comment by vicroms** [comment]
We should test all ports that use `find_acquire_program(PYTHON3|PYTHON2)`

File: ports/duktape/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Source: duktape
-Version: 2.4.0-6
-Homepage: https://github.com/svaarala/duktape
-Description: Embeddable Javascript engine with a focus on portability and compact footprint.
+Source: duktape
+Version: 2.4.0-6
```
Comment (PhoebeHui) [inline ports/duktape/CONTROL]: Could you update the Version to 2.4.0-7

**Review by PhoebeHui — CHANGES_REQUESTED** [review]

**Comment by PhoebeHui** [comment]
The CI testing failed, could you update the ci baseline file?

##[error] REGRESSION: shiva-sfml:x64-windows
##[error] PASSING, REMOVE FROM FAIL LIST: shiva:x64-windows (C:\vsts\_work\3\s\scripts\ci.baseline.txt)

File: ports/duktape/portfile.cmake [hunk 2]
```diff
@@ -16,7 +16,7 @@ file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/duktapeConfig.cmake.in DESTINATION ${SOURCE_PATH})
```
Comment (NancyLi1013) [inline ports/duktape/portfile.cmake]: `include(vcpkg_common_functions)` is no longer needed.
Could you please remove this line?

Comment (NancyLi1013) [reply ports/duktape/portfile.cmake]: Could you update` VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux"` as `VCPKG_TARGET_IS_LINUX`?

File: ports/shiva/CONTROL [hunk 3]
```diff
@@ -1,4 +1,4 @@
 Source: shiva
-Version: 1.0-2
+Version: 1.0-3
 Description: Modern C++ Game Engine
 Build-Depends: boost-stacktrace, boost-filesystem, boost-dll, entt, lua, luafilesystem (windows), sol2, pybind11, spdlog, nlohmann-json
```
Comment (NancyLi1013) [inline ports/shiva/CONTROL]: Could you add `homepage` here?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by JackBoosY** [comment]
> CMake Error at D:/downloads/tools/cmake-3.14.0-windows/cmake-3.14.0-win32-x86/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find PythonInterp (missing: PYTHON_EXECUTABLE)
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.14.0-windows/cmake-3.14.0-win32-x86/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  D:/downloads/tools/cmake-3.14.0-windows/cmake-3.14.0-win32-x86/share/cmake-3.14/Modules/FindPythonInterp.cmake:160 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  C:/vsts/_work/4/s/scripts/buildsystems/vcpkg.cmake:285 (_find_package)
  C:/vsts/_work/4/s/installed/x64-windows/share/cmake/pybind11/FindPythonLibsNew.cmake:60 (find_package)
  C:/vsts/_work/4/s/scripts/buildsystems/vcpkg.cmake:285 (_find_package)
  C:/vsts/_work/4/s/installed/x64-windows/share/cmake/pybind11/pybind11Tools.cmake:16 (find_package)
  C:/vsts/_work/4/s/installed/x64-windows/share/cmake/pybind11/pybind11Config.cmake:100 (include)
  C:/vsts/_work/4/s/scripts/buildsystems/vcpkg.cmake:285 (_find_package)
  modules/python/CMakeLists.txt:4 (find_package)

**Comment by JackBoosY** [comment]
Since shiva-sfml does not match the dependency entt version, regression will be resolved in the next update of shiva-sfml.

**Pull request merged**